src/llvm/ext/llvm_ext.cc

```diff
@@ -6,6 +6,10 @@
 #include <llvm/IR/DebugLoc.h>
 #include <llvm/IR/LLVMContext.h>
 #include <llvm/IR/Metadata.h>
+#include "llvm/Analysis/ModuleSummaryAnalysis.h"
```

Comment:
Well, it's not the same, since `#include <...>` form uses system header files, while `#include "..."` starts from current directory. See http://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename for reference.